### DISCUSSION TITLE
Add links to categories and keywords listings on front page and/or "Browse All Crates"

### DISCUSSION THREAD

**Discussion by jwodder** [rust-lang/crates.io#4958] [Feature Requests]
The pages https://crates.io/categories and https://crates.io/keywords (and possibly others I haven't found) seem immensely useful for navigating crates by related topics, yet there are no links to these pages on either the front page of crates.io or on the ["Browse All Crates"](https://crates.io/crates) page, where they would be most welcome.  (https://crates.io/category_slugs would be another good page to link somewhere; it currently only seems to be linked by a chapter of the Cargo Book.)

Compare [PyPI's browse/search page](https://pypi.org/search/), which features links to its classifiers on the left, including a link to [a page of all classifiers](https://pypi.org/classifiers/).

I therefore suggest adding explicit links to these pages somewhere on the front page of crates.io and/or on the "Browes All Crates" page.

**Comment by Turbo87** [comment]
<img width="620" alt="Bildschirmfoto 2022-07-07 um 17 21 36" src="https://user-images.githubusercontent.com/141300/177810596-f95b8f65-976e-43bd-b58d-fb8be239f70b.png">

the "see all" links there link to exactly those two pages 😅

**Reply by jwodder** [reply]
Well, those should be more prominent!  (And they didn't even show up when I searched the page source for "keywords" and "categories".)
